packages/flutter_tools/lib/src/base/utils.dart

```diff
@@ -24,21 +24,24 @@ class BotDetector {
     return platform.environment['BOT'] != 'false'
        && (platform.environment['BOT'] == 'true'
 
-           // https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables
+        // https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables
```

Comment:
We removed travis and appveyor in other places. Why don't we remove them here?

Reply:
Because it is an all-purpose detection function, not just specific to this repo, and the flutter tool may run on other repos that need to detect this.  For instance, the plugins repo uses flutter and still uses Travis.